app/models/fe/element.rb

```diff
@@ -119,7 +119,7 @@ def previous_element(question_sheet, page = nil)
     end
 
     def hidden_by_conditional?(answer_sheet, page)
-      prev_el = previous_element(answer_sheet.question_sheet, page)
+      prev_el = previous_element(page.question_sheet, page) if answer_sheet.question_sheets.include?(page.question_sheet)
```

Comment:
Hmm I don't think this would work if the if is false.  prev_el would be nil then, and the next line does calls o nit.

I think we can actually leave this line how it was, since the method itself requires a page to be passed in.

But I think https://github.com/CruGlobal/fe/blob/fix_for_mult_question_sheets/app/models/fe/element.rb#L95 should read

```
    def previous_element(question_sheets, page = nil)
```

and then in that method, https://github.com/CruGlobal/fe/blob/fix_for_mult_question_sheets/app/models/fe/element.rb#L97

```
page ||= pages_on.detect{ |p| question_sheets.include?(p.question_sheet) }
```